app/code/GorbanSv/AskQuestion/Observer/Model/ResourceModel/LoadBefore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace GorbanSv\AskQuestion\Observer\Model\ResourceModel;
+
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+
+/**
+ * Class LoadBefore
+ * @package GorbanSv\AskQuestion\Observer\Model\ResourceModel
+ */
+class LoadBefore implements ObserverInterface
+{
+    /**
+     * @param Observer $observer
+     */
+    public function execute(Observer $observer)
+    {
+        $data = $observer->getEvent()->getData();
```

Comment:
This observer does not perform any actions. What actions are planned? To update modification time, set the status if empty?

Reply:
... if no action is planned, please remove unnecessary observers